connection.go

```diff
@@ -518,7 +518,9 @@ func (c *Connection) Reply(ctx context.Context, opErr error) error {
 		if opErr == nil {
 			c.debugLog(fuseID, 1, "-> %s", describeResponse(op))
 		} else {
-			c.debugLog(fuseID, 1, "-> Error: %q", opErr.Error())
+			if !c.shouldLogError(op, opErr) {
```

Comment:
With your suggestion, we’re calling shouldLogError twice when both error and debug logging is enabled, which is unnecessary. Please store the result in a variable, e.g.:

```go
logError := c.shouldLogError(op, opErr)
if c.debugLogger != nil {
  // …
}

// Error logging
if logError {
  // …
}
```

Reply:
Yep. Done.